aws/wafv2_helper.go

```diff
@@ -862,20 +1069,107 @@ func flattenWafv2RuleAction(a *wafv2.RuleAction) interface{} {
 	m := map[string]interface{}{}
 
 	if a.Allow != nil {
-		m["allow"] = make([]map[string]interface{}, 1)
+		m["allow"] = flattenWafv2Allow(a.Allow)
 	}
 
 	if a.Block != nil {
-		m["block"] = make([]map[string]interface{}, 1)
+		m["block"] = flattenWafv2Block(a.Block)
 	}
 
 	if a.Count != nil {
-		m["count"] = make([]map[string]interface{}, 1)
+		m["count"] = flattenWafv2Count(a.Count)
 	}
 
 	return []interface{}{m}
 }
 
+func flattenWafv2Allow(a *wafv2.AllowAction) interface{} {
+	if a == nil {
+		return map[string]interface{}{}
```

Comment:
Here we'll want to use the same array of interfaces as the final return statement (in this case empty)
```suggestion
		return []interface{}{}
```